quantus/helpers/tf_model.py

```diff
@@ -93,3 +94,29 @@ def get_random_layer_generator(self, order: str = "top_down", seed: int = 42):
             np.random.seed(seed=seed + 1)
             layer.set_weights([np.random.permutation(w) for w in weights])
             yield layer.name, random_layer_model
+
+    def get_hidden_layers_outputs(self,
+                                  x: np.ndarray,
+                                  layer_names: Optional[List[str]] = None,
+                                  layer_indices: Optional[List[int]] = None,
+                                  ) -> np.ndarray:
+        batch_size = x.shape[0]
```

Comment:
Docstring

Reply:
there's one in base class https://github.com/aaarrti/Quantus/blob/relative-stability/quantus/helpers/model_interface.py#L53-L68.